File: Client/Electron/transferModel/transferModel.js
export class TransferModel {
  constructor(data, actionType) {
    this.ActionType = actionType;
    this.TransferObject = JSON.stringify(data);
  }
  get getReponse() {
    return {
      ...this.ActionType,
      ...this.TranferObject,
    };
  }
  toString() {
    JSON.stringify({ ...this.ActionType, ...this.TransferObject });
  }
}

File: Server/src/main/java/Models/TransferModel.java
package Models;

public class TransferModel {
    public int ActionType;
    public Object TransferObject;
    public TransferModel(){};
    public TransferModel(int actionType, Object transferObject) {
        ActionType = actionType;
        TransferObject = transferObject;
    }

    @Override
    public String toString() {
        return "TransferModel{" +
                "ActionType=" + ActionType +
                ", TransferObject='" + TransferObject + '\'' +
                '}';
    }
}
